packages/block-editor/src/components/block-mobile-toolbar/index.native.js

```diff
@@ -48,13 +56,15 @@ export default compose(
 			order: getBlockIndex( clientId ),
 		};
 	} ),
-	withDispatch( ( dispatch, { clientId, rootClientId } ) => {
+	withDispatch( ( dispatch, { clientId, rootClientId, onDeleteBlock } ) => {
```

Comment:
Why not just `onDelete`? I think it is like thin in Columns PR - https://github.com/WordPress/gutenberg/pull/19013/files#diff-b20ce6a53833ca6ec70166ea994cb910R59